api/src/main/java/org/pmiops/workbench/cdr/CdrDbConfig.java

```diff
@@ -74,15 +73,15 @@ public CdrDataSource(CdrVersionDao cdrVersionDao,
         if (cdrVersion.getName().equals(workbenchConfig.cdr.defaultCdrVersion)) {
           cdrVersionId = cdrVersion.getCdrVersionId();
         }
-        // The database name used for a given CDR version is depends on whether this is the
-        // workbench database user or data browser user.
-        String dbName = isWorkbenchDbUser ? cdrVersion.getCdrDbName() : cdrVersion.getPublicDbName();
+
+        // TODO: Reconcile this with the public-api DB configs; this should
```

Comment:
So for now, public is still pointed at a fixed public DB... and in a follow-up we'll make it use this sort of logic?

(SGTM.)

Reply:
Yep, exactly.